sunpy/net/hek2vso/hek2vso.py

```diff
@@ -51,10 +51,10 @@ def translate_results_to_query(results):
ormat='isot' value=2011-08-09T07:27:00.000>, None)>, <Source('SDO')>, <Instrument('AIA')>, <Wavelength(93.99999999999999, 93.99999999999999, 'Angstrom')>], [<Time(<Time object: scale='utc' format='isot'value=2011-08-09T07:19:00.000>, <Time object: scale='utc' format='isot' value=2011-08-09T07:27:00.000>, None)>, <Source('GOES')>, <Instrument('GOES')>, <Wavelength(4.999999999999999, 4.999999999999999, 'Angstrom')>], [<Time(<Time object: scale='utc' format='isot' value=2011-08-09T07:22:38.000>, <Time object: scale='utc' format='isot' value=2011-08-09T08:32:02.000>, None)>, <Source('SDO')>,<Instrument('AIA')>, <Wavelength(210.99999999999997, 210.99999999999997, 'Angstrom')>], [<Time(<Time object: scale='utc' format='isot' value=2011-08-09T07:22:44.000>, <Time object: scale='utc' format='isot' value=2011-08-09T07:28:56.000>, None)>, <Source('SDO')>, <Instrument('AIA')>, <Wavelength(193.0, 193.0, 'Angstrom')>], [<Time(<Time object: scale='utc' format='isot' value=2011-08-09T07:48:00.000>, <Time object: scale='utc' format='isot' value=2011-08-09T08:08:00.000>, None)>, <Source('SDO')>, <Instrument('AIA')>, <Wavelength(93.99999999999999, 93.99999999999999, 'Angstrom')>], [<Time(<Time object: scale='utc' format='isot' value=2011-08-09T07:48:00.000>, <Time object: scale='utc' format='isot' value=2011-08-09T08:08:00.000>, None)>, <Source('GOES')>, <Instrument('GOES')>, <Wavelength(4.999999999999999, 4.999999999999999, 'Angstrom')>], [<Time(<Time object: scale='utc' format='isot' value=2011-08-09T07:55:59.000>, <Time object: scale='utc' format='isot' value=2011-08-09T08:28:47.000>, None)>, <Source('SDO')>, <Instrument('AIA')>, <Wavelength(130.99999999999997, 130.99999999999997, 'Angstrom')>], [<Time(<Time object: scale='utc' format='isot' value=2011-08-09T07:59:49.000>, <Time object: scale='utc' format='isot' value=2011-08-09T08:25:37.000>, None)>, <Source('SDO')>, <Instrument('AIA')>, <Wavelength(170.99999999999997, 170.99999999999997, 'Angstrom')>], [<Time(<Time object: scale='utc' format='isot' value=2011-08-09T08:00:03.000>, <Time object: scale='utc' format='isot' value=2011-08-09T08:15:15.000>, None)>, <Source('SDO')>, <Instrument('AIA')>, <Wavelength(93.99999999999999, 93.99999999999999, 'Angstrom')>], [<Time(<Time object: scale='utc' format='isot' value=2011-08-09T08:00:20.000>, <Time object: scale='utc' format='isot' value=2011-08-09T08:28:44.000>, None)>, <Source('SDO')>, <Instrument('AIA')>, <Wavelength(193.0, 193.0, 'Angstrom')>], [<Time(<Time object: scale='utc' format='isot' value=2011-08-09T08:00:53.000>, <Timeobject: scale='utc' format='isot' value=2011-08-09T08:15:53.000>, None)>, <Source('SDO')>, <Instrument('AIA')>, <Wavelength(334.99999999999994, 334.99999999999994, 'Angstrom')>], [<Time(<Time object: scale='utc' format='isot' value=2011-08-09T08:01:21.000>, <Time object: scale='utc' format='isot' value=2011-08-09T08:16:45.000>, None)>, <Source('SDO')>, <Instrument('AIA')>, <Wavelength(303.99999999999994, 303.99999999999994, 'Angstrom')>]]
```

Comment:
Can you put `...` in here to shorten it like it is in the old version? (doctests should still pass.)